Makefile

```diff
@@ -83,3 +88,4 @@ clean:
 	$(lint-targets) lint \
 	$(test-targets) test \
 	clean
+	windows
```

Comment:
Missing backslash after `clean`.
`windows` should be called `build-windows` since that's the name of the rule.